pages/stores/index.js

```diff
@@ -3,26 +3,55 @@ import Layout from '../../components/layout'
 import Navbar from '../../components/navbar'
 import { StoreCard } from '../../components/store/card'
 import { getStores } from '../../data/stores'
+import { getProductByCustomerId, getProducts } from '../../data/products'
+import { ProductCard } from '../../components/product/card'
+import { IndexStoreCard } from '../../components/store/index-card'
 
 
 export default function Stores() {
   const [stores, setStores] = useState([])
+  const [products, setProducts] = useState([])
 
   useEffect(() => {
-    getStores().then(data => {
+    let activeStores = []
+    getStores().then(stores => {
+      if (stores) {
+        for (let store of stores) {
+          getProductByCustomerId(store.owner.id).then(data => {
+            if (data.length != 0)
+            store.productcount = data.length
```

Comment:
I think you dropped some curly braces here. Lines 22-23 should be wrapped in curly braces to be executed with the if statement on line 21.